templates/resources/address-space-controller/address-space-definitions.yaml

```diff
@@ -143,7 +143,7 @@ data:
                 value: '55671'
               - name: CERT_DIR
                 value: /etc/enmasse-certs
-              image: rh-amqmaas-1/amqmaas10-router-metrics-openshift:latest
+              image: /rh-amqmaas-1/amqmaas10-router-metrics-openshift:latest
```

Comment:
I'll fix. The script seemed to add a leading slash to all the image names. I'll check why.  I take it that none of the images should have a leading slash.

Reply:
@k-wall The format is $DOCKER_REGISTRY/$DOCKER_ORG/$COMPONENT . I think that we should fix the scripts upstream to handle a missing $DOCKER_REGISTRY, because that will allow these templates to work with the Red Hat container registry without changing them.